app/models/event.rb

```diff
@@ -0,0 +1,9 @@
+class Event < ActiveRecord::Base
+  has_many :user_events
+  has_many :users, through: :user_events
+  belongs_to :location
+
+  accepts_nested_attributes_for :users, allow_destroy: true
```

Comment:
allow_destroy option allow to delete object from db. that's definitely not what we want to do.